funnel/templates/layout.html.jinja2

```diff
@@ -5,9 +5,12 @@
   <title>{% block titleblock %}{% block title %}{{ title }}{% endblock %}{% if title_suffix %} – {{ title_suffix }}{% elif self.title() != config['SITE_TITLE'] %} – {{ config['SITE_TITLE'] }}{% endif %}{%- endblock %}</title>
   <meta name="DC.title" content="{{ self.titleblock()|e }}" />
   <meta property="og:title" content="{{ self.titleblock()|e }}" />
-  <meta property="og:type" content="website" />
+  {%- block homescreen %}
+  {%- if not project %}
   <meta property="og:url" content="{{ url_for('index', _external=true) }}" />
```

Comment:
Removing this line will do.
`baseframe.html.jinja2` template takes care of adding the url https://github.com/hasgeek/baseframe/blob/master/baseframe/templates/baseframe/mui/baseframe.html.jinja2#L33

Reply:
Done!